tests/remote/pusher/keepalive.js

```diff
@@ -8,22 +10,31 @@ describe("Pusher (integration)", function () {
 
     beforeEach(function () {
       pusher = new Pusher.forURL(process.env.PUSHER_URL, {
-        keepAlive: true,
+        agent: new https.Agent({ keepAlive: true }),
       })
     })
 
+    // TODO this test is useless. It passes if keepAlive is false...
```

Comment:
Yeah, I don't really understand what this is trying to test 🤔

Reply:
In case it wasn't obvious, I reviewing this commit by commit, feel free to ignore this. I see you removed it in a subsequent commit